src/WinUI/ArcGIS.WinUI.Viewer/SamplePage.xaml

```diff
@@ -1,59 +1,83 @@
-﻿<Page x:Class="ArcGIS.WinUI.Viewer.SamplePage"
-      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-      xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
-      xmlns:local="using:ArcGIS.WinUI.Viewer">
-    <TabView x:Name="Tabs"
-             HorizontalAlignment="Stretch"
-             VerticalAlignment="Stretch"
-             CanDragTabs="False"
-             CanReorderTabs="False"
-             IsAddTabButtonVisible="False">
+﻿<Page
```

Comment:
Can you double check your XAML styler config? The git diff shouldn't be this huge.